match.go

```diff
@@ -41,10 +41,7 @@ import (
 	"strings"
 )
 
-var (
-	goroot       = filepath.Clean(runtime.GOROOT())
-	gorootSrcPkg = filepath.Join(goroot, "src/pkg")
-)
+var goroot = filepath.Clean(runtime.GOROOT())
```

Comment:
The `goroot` variable is only used in `go1.3.go` and `go1.4.go`, it should probably be moved to these files respectively.